src/components/HomePageContent/HowItWorksSection/Steps.tsx

```diff
@@ -36,7 +36,7 @@ export const Steps: React.FunctionComponent<StepsProps> = ({ contentType, stepsD
       {contentType !== ContentType.BENEFIT && (
         <h3 className="hidden text-lemon-500 text-center mt-12 lg:inline">{contentType}</h3>
       )}
-      <div className="flex flex-wrap m-auto lg:px-16 2xl:px-0 items-start lg:w-full 2xl:max-w-7xl">
+      <div className="flex flex-wrap m-auto 2xl:px-0 items-start lg:w-full 2xl:max-w-7xl">
         {stepsDetails.map((stepDetail: StepDetail, index: number) => (
```

Comment:
same feedback as above.

Reply:
ok for this, i revert back cos i dont think need this padding anymore